src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,13 +1,19 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Truck
- * Do not remove no-field constructor
- */
 public class Truck extends Machine {
+    private int curbWeight;
```

Comment:
The field `curbWeight` should be private to prevent direct access to the field. This is to ensure the field is only accessible through its getter and setter, which provides better encapsulation.